tests/vmcheck/test-override-local-replace.sh

```diff
@@ -164,17 +164,22 @@ echo "ok active -> inactive -> active override replace"
 
 vm_rpmostree cleanup -p
 
-# try both local package layering and local replacements to
-# make sure fd sending doesn't get mixed up
+# try both local package layering and local replacements to make sure fd sending
+# doesn't get mixed up; and also remove a package at the same time
 vm_build_rpm baz
 vm_cmd ostree commit -b vmcheck --tree=ref=vmcheck_tmp/with_foo_and_bar
 vm_rpmostree upgrade
 vm_rpmostree override replace $YUMREPO/bar-0.9-1.x86_64.rpm \
-                       --install $YUMREPO/baz-1.0-1.x86_64.rpm
+                       --install $YUMREPO/baz-1.0-1.x86_64.rpm \
+                       --remove foo
```

Comment:
Looks like we may need `--remove fooext` here too?

Reply:
Ahh, right. Fixup below! ⬇️